xwiki-platform-core/xwiki-platform-mentions/xwiki-platform-mentions-notifications/pom.xml

```diff
@@ -73,19 +73,14 @@
     </dependency>
     <dependency>
       <groupId>org.xwiki.platform</groupId>
-      <artifactId>xwiki-platform-web-templates</artifactId>
-      <version>${project.version}</version>
-      <scope>test</scope>
-    </dependency>
-    <dependency>
-      <groupId>org.xwiki.platform</groupId>
-      <artifactId>xwiki-platform-icon-script</artifactId>
+      <artifactId>xwiki-platform-date</artifactId>
       <version>${project.version}</version>
       <scope>test</scope>
```

Comment:
I don't understand this scope. Why is it for testing only?

Reply:
Should all the script services used in the templates of a module be added to the module's dependencies?
If that's the case, then I must remove the test scope for this one and add some other dependencies (icons + localization).

Reply:
What I meant is that if the vm uses the icon script service, then surely this is a runtime depenency and thus needs to be added to the module that has that template, and not as a "test" scope.